docs/en/releases/index.md

```diff
@@ -2,7 +2,8 @@
 
 A list of PX4 release notes, they contain a list of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
 
-- [main](../releases/main.md) (changes since v1.15)
+- [main](../releases/main.md) (changes since v1.16)
```

Comment:
Re https://github.com/PX4/PX4-Autopilot/pull/24793/files#r2076602611

We should perhaps delete the doc in this branch.

Reply:
In the main branch, we could then delete everything too, except a link to the built version of v1.16 docs "for now" until that gets released.